sos/report/plugins/cups.py

```diff
@@ -18,6 +20,37 @@ class Cups(Plugin, IndependentPlugin):
 
     packages = ('cups',)
 
+    def get_jids(self, dir="/var/spool/cups", n_jids=10,
+                 exists_f=os.path.exists, listdir_f=os.listdir):
+        jids = []
+
+        if not exists_f(dir):
+            return
+
+        spool_files = listdir_f(dir)
```

Comment:
I'm not sure why we're passing `os.path.exists` or `os.listdir` as arguments just to use them like this? Just call the `os` methods directly.

Reply:
It is for possible mocking of functions during testing - AFAIK the convention is the testing should not touch files on the system where the tests are run, so this way you can pass a fake function, which you will implement in the test - it will accept the same args as os methods and will give the same output.